app/herdbook.py

```diff
@@ -238,33 +241,82 @@ def individual(i_id):
     """
     user_id = session.get("user_id", None)
     ind = da.get_individual(i_id, user_id)
+
     if ind:
-        ind["inbreeding"] = "%.2f" % (get_inbreeding(ind['id']) * 100)
+        ind["inbreeding"] = "%.2f" % (get_inbreeding(ind['id'], ind['genebank_id']) * 100)
```

Comment:
```suggestion
        ind["inbreeding"] = "%.2f" % (get_inbreeding(ind['number'], ind['genebank_id']) * 100)
        ind["MK"] = "%.2f" % (get_mean_kinship(ind['number'], ind['genebank_id']) * 100)
```
Need to use the Individual Number instead of the database ID 
Add the Mean Kinship value to the ind dict